frontend/Components/ListingCard/ListingCard.module.css

```
@@ -0,0 +1,41 @@
+.cardContainer {
+  width: 350px;
+}
+
+.cardContent {
+  display: flex;
+  flex-direction: column;
+  gap: 1.3em;
+}
+
+.cardHeader {
```

Ya que usamos CSS modules, esto se puede llamar:

```suggestion
.header {
```